docs/source/API/reference.rst

```diff
@@ -35,6 +36,18 @@ Functions
 
    asreview.init_sampling.sample_prior_knowledge
    asreview.search.fuzzy_find
+   asreview.types.type_n_queries
+   asreview.utils.asreview_path
+   asreview.utils.get_data_home
+   asreview.utils.format_to_str
+   asreview.utils.pretty_format
+   asreview.utils.is_iterable
+   asreview.utils.list_model_names
+   asreview.utils.list_reader_names
+   asreview.utils.list_writer_names
+   asreview.utils.get_entry_points
+   asreview.utils.is_url
+   asreview.utils.get_random_state
```

Comment:
```suggestion
   asreview.utils.asreview_path
   asreview.utils.get_data_home
   asreview.utils.list_model_names
   asreview.utils.list_reader_names
   asreview.utils.list_writer_names
   asreview.utils.get_entry_points
```

I think this should be added to a category misc or utils. This is not top level.